src/styles/blocks/action.scss

```
@@ -0,0 +1,29 @@
+.action {
+  &__link {
+    display: none;
+    @include whiteLink;
+    text-transform: uppercase;
+    position: relative;
+
+    @include onDesktop {
+      @include bottomBorder;
+    }
+  }
+
+  &__icon {
+    cursor: pointer;
+    transition: transform 300ms;
```

$duration: 300ms;
```suggestion
    transition: transform $duration;
```

если завтра захочешь чтобы везде было не 300 а 400, то можно будет просто изменить в переменной